cmd/esc/cli/cli_test.go

```diff
@@ -627,6 +627,86 @@ func (c *testPulumiClient) GetOpenProperty(ctx context.Context, orgName, envName
 	return nil, errors.New("NYI")
 }
 
+func (c *testPulumiClient) GetEnvironmentTag(
```

Comment:
Can you add some tests?

If you add a new file in `cli/testdata`, it will be automatically picked up by the test harness:

- The `run` section contains the command(s) to run
- The `environments` section contains the environments to define for testing

Once you've filled in those two sections, you can generate the `stdout`/`stderr` sections by running `PULUMI_ACCEPT=1 go test ./cmd/esc/cli` from the repository root.

You'll likely need to revert changes to `cmd/esc/cli/testdata/env-login-gh100.yaml` and `cmd/esc/cli/testdata/env-set.yaml` as the regeneration process makes undesirable formatting changes to those files.